docker/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+# Copy the config files into the Docker folder
```

Comment:
What do you think about just automatically `cd`ing into the correct directory rather than forcing the user to invoke from `docker/`?